pkg/parser/seven/parser.go

```diff
@@ -216,6 +218,17 @@ func decodeVerdict(verdict accesslog.FlowVerdict) pb.Verdict {
 	}
 }
 
+func decodeTrafficDirection(direction accesslog.ObservationPoint) pb.TrafficDirection {
+	switch direction {
+	case accesslog.Ingress:
+		return pb.TrafficDirection_INGRESS
+	case accesslog.Egress:
+		return pb.TrafficDirection_EGRESS
+	default:
+		return pb.TrafficDirection_TRAFFIC_DIRECTION_UNKNOWN
```

Comment:
:O

Reply:
The naming is not great, but that's due to protobuf not having namespaced enums. This requres us to prefix `UNKNOWN`s at the protobuf level, otherwise it would collide with other enums that have an `UNKNOWN` variant :man_shrugging: